lib/Makefile

```diff
@@ -140,6 +144,8 @@ PREFIX      ?= /usr/local
 prefix      ?= $(PREFIX)
 EXEC_PREFIX ?= $(prefix)
 exec_prefix ?= $(EXEC_PREFIX)
+BINDIR      ?= $(exec_prefix)/bin
```

Comment:
It seems you need to install libraries into `/bin` rather than `/lib`, presumably due to `PATH` constraints.
OK, fine.
Just add a code comment here to explain why `BINDIR` exists, so that it doesn't get misused or wrongly modified by some future developer.

Reply:
The DLL are indeed usually installed in the bin/ directory. That is where build systems (autotools with libtool, cmake, meson) are installing DLL, contrary to UNIX where shared lib are in lib/
The import lib (libz4.lib) is usually installed in the lib/ directory

Also note the name of the DLL : libz4-1.dll This number is the major version of the library. I also follow what other build systems do.

I will add the comment in lib/Makefile

should I also update README.md, and if yes which one ? top level or lib/README.md ?

Reply:
There is very little in term of doc in this case,
because it's not about providing a control that user must be aware of for the functionality to work.
Here, everything is transparent, and the objective is: `MSYS2` is now a valid environment for `make install` target.

In theory, it would be useful to document that `MSYS2` is a new valid environment,
but since we don't have such list within any `README`, there is no place for this statement.

Except maybe the `Makefile` itself (see line 131).